src/Redmine/Client.php

```diff
@@ -364,12 +314,7 @@ public function getCheckSslCertificate()
      */
     public function setCheckSslHost($check = false)
```

Comment:
```suggestion
    public function setCheckSslHost(bool $check = false)
```

Reply:
Changing types for public classes would be a breaking change.

Reply:
It is possible that someone called `$client->setCheckSslHost(1)`. This would be break.